get-started/swarm-deploy.md

```diff
@@ -1,47 +1,48 @@
 description: Learn how to describe and deploy a simple application on Docker Swarm.
+redirect_from:
+- /get-started/part4/
 ---
-{% include_relative nav.html selected="4" %}
 
 ## Prerequisites
 
+- Download and install Docker Desktop as described in [Orientation and setup](/index.md).
 - Work through containerizing an application in [Part 2](part2.md).
 - Make sure that Swarm is enabled on your Docker Desktop by typing `docker system info`, and looking for a message `Swarm: active` (you might have to scroll up a little).
 
-  If Swarm isn't running, simply type `docker swarm init` at a shell prompt to set it up.
+  If Swarm isn't running, simply type `docker swarm init` in a shell prompt to set it up.
 
 ## Introduction
 
 Now that we've demonstrated that the individual components of our application run as stand-alone containers and shown how to deploy it using Kubernetes, let's look at how to arrange for them to be managed by Docker Swarm. Swarm provides many tools for scaling, networking, securing and maintaining your containerized applications, above and beyond the abilities of containers themselves.
 
 In order to validate that our containerized application works well on Swarm, we'll use Docker Desktop's built in Swarm environment right on our development machine to deploy our application, before handing it off to run on a full Swarm cluster in production. The Swarm environment created by Docker Desktop is _fully featured_, meaning it has all the Swarm features your app will enjoy on a real cluster, accessible from the convenience of your development machine.
 
-## Describing Apps Using Stack Files
+## Describe apps using stack files
 
-Swarm never creates individual containers like we did in the previous step of this tutorial; instead, all Swarm workloads are scheduled as _services_, which are scalable groups of containers with added networking features maintained automatically by Swarm. Furthermore, all Swarm objects can and should be described in manifests called _stack files_; these YAML files describe all the components and configurations of your Swarm app, and can be used to easily create and destroy your app in any Swarm environment.
+Swarm never creates individual containers like we did in the previous step of this tutorial. Instead, all Swarm workloads are scheduled as _services_, which are scalable groups of containers with added networking features maintained automatically by Swarm. Furthermore, all Swarm objects can and should be described in manifests called _stack files_. These YAML files describe all the components and configurations of your Swarm app, and can be used to easily create and destroy your app in any Swarm environment.
 
-1.  Let's write a simple stack file to run and manage our bulletin board. Place the following in a file called `bb-stack.yaml`:
+1. Let's write a simple stack file to run and manage our bulletin board. Place the following in a file called `bb-stack.yaml`:
 
     ```yaml
```

Comment:
I just tried to copy yaml from the rendered html file. From line two each line gets an extra space at the beginning of each line. That corrupts the yaml.

Reply:
This should be fixed now. Could you try copying the yaml file again?